mjcf_to_sdformat/mjcf_to_sdformat/converters/link.py

```diff
@@ -96,34 +99,65 @@ def mjcf_geom_to_sdf(body):
 
     link.set_raw_pose(su.get_pose_from_mjcf(body))
 
+    def get_orientation(geom):
+        """
+        Get orientation from a MJCF geom when it's defined with "fromto"
+
+        :param mjcf.Element geom: MJCF geom to extract the orientation
+        :return: The newly created quaterion.
+        :rtype ignition.math.Quateriond
+        """
+        if geom.fromto is not None:
+            v1 = Vector3d(geom.fromto[0], geom.fromto[1], geom.fromto[2])
+            v2 = Vector3d(geom.fromto[3], geom.fromto[4], geom.fromto[5])
+            vec = (v1 - v2).normalize()
+            z = Vector3d(0, 0, 1)
+            quat = Quaterniond()
+            quat.set_from_2_axes(z, vec)
+            return quat
+        return Quaterniond()
+
+    def get_position(geom):
+        """
+        Get the translattion from a MJCF geom when it's defined with "fromto"
+
+        :param mjcf.Element geom: MJCF geom to extract the position
+        :return: The newly created Vector3d.
+        :rtype ignition.math.Vector3d
+        """
+        if geom.fromto is not None:
+            v1 = Vector3d(geom.fromto[0], geom.fromto[1], geom.fromto[2])
+            v2 = Vector3d(geom.fromto[3], geom.fromto[4], geom.fromto[5])
+            return (v1 + v2) / 2.0
+        return Vector3d()
+
+    def set_visual(geom):
+        visual = mjcf_visual_to_sdf(geom)
+        if visual is not None:
+            visual.set_name(su.prefix_name_with_index(
+                "visual", geom.name, NUMBER_OF_VISUAL))
+            pose_form_to = Pose3d(get_position(geom), get_orientation(geom))
+            pose = pose_form_to * su.get_pose_from_mjcf(geom)
```

Comment:
@azeey Fixed here: https://github.com/gazebosim/gz-mujoco/pull/47/commits/38c9c9a8fa93113c3c9802f1e8b88d9869691cbd

Reply:
It's true, but this will allow to handle both situations when `fromto` is defined or when `pos` + `euler`

Reply:
If we do ` pose = pose_form_to * su.get_pose_from_mjcf(geom)` and the geom has both `euler` and `fromto`, the result would be incorrect because `pose` will now have rotation from both `euler` and `fromto`. Instead, i think we need to check if `fromto` is defined and set `pose` to the position/rotation only from `fromto`, otherwise set `pose` to the position/rotation from `pos` and `euler`.

Reply:
https://github.com/gazebosim/gz-mujoco/pull/47/commits/bb0d08d71191d4817f026d9ca1840079c7a5beb3